src/gossip/pull_request.zig

```diff
@@ -16,7 +16,7 @@ const RwMux = sig.sync.RwMux;
 const exp = std.math.exp;
 
 pub const MAX_BLOOM_SIZE: usize = 928;
-pub const MAX_NUM_PULL_REQUESTS: usize = 20; // labs - 1024;
+pub const MAX_NUM_PULL_REQUESTS: usize = 256;
```

Comment:
this looks like the corresponding constant in Agave https://github.com/anza-xyz/agave/blob/37f6e9ee9ebaff2251d7d205092c508521345c28/gossip/src/crds_gossip_pull.rs#L172 and it is set much higher. Any reason not to match the same value?